### PR TITLE
Changes to support both Mysql  5 and 8 version 

### PR DESCRIPTION
**Pull request by jatinbajaj1**
In .env 
for Mysql 8
MYSQL_VER=8
MYSQL_TAG=latest

for Mysql 5 
MYSQL_VER=5
MYSQL_TAG=5.6

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -7,7 +7,7 @@ networks:
       - subnet: ${SUBNET}
 services:
   database:
-    image: mysql/mysql-server:5.6
+    image: mysql/mysql-server:8.0.21
```
Comment (simingy) [inline docker-compose.yml:10]: i think 8.0 is ok

File: etc/my.cnf [hunk 2]
```diff
@@ -27,6 +27,6 @@ user=mysql
 # Disabling symbolic-links is recommended to prevent assorted security risks
 symbolic-links=0
 
-log-error=/var/log/mysqld.log
+#log-error=/var/log/mysqld.log
```
Comment (simingy) [inline etc/my.cnf:30]: intended?

Comment (jatinbajaj1) [reply etc/my.cnf:30]: Actually in the   "latest" Tag image  
when we bring up the container  it complains with the error:
'/var/lib/mysqld.log'   permission denied,    that was in the latest tag  
seems like its working in  8.0
i would change this

File: docker-compose.yml (hunk 1 above)
Comment (jatinbajaj1) [reply docker-compose.yml:10]: sure ,  would change

File: etc/my.cnf (hunk 2 above)
Comment (jatinbajaj1) [reply etc/my.cnf]: changes added

File: docker-compose.yml (hunk 1 above)
Comment (jatinbajaj1) [reply docker-compose.yml]: changes added

**Review by simingy — APPROVED** [review]

**Pull request merged**